query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/data_types/decimal.rs

```diff
@@ -10,17 +10,15 @@ mod decimal {
         let schema = indoc! {
             r#"model Model {
               #id(id, Int, @id)
-              field Decimal? @default("1.00112233445566778899")
+              field Decimal? @default("1.001122334455668")
```

Comment:
Is there any reason we don't send decimals as strings to JS like this?

```diff
diff --git a/query-engine/driver-adapters/src/conversion/postgres.rs b/query-engine/driver-adapters/src/conversion/postgres.rs
index 77e79f549d..21b1ec6b2f 100644
--- a/query-engine/driver-adapters/src/conversion/postgres.rs
+++ b/query-engine/driver-adapters/src/conversion/postgres.rs
@@ -37,11 +37,8 @@ pub fn values_to_js_args(values: &[quaint::Value<'_>]) -> serde_json::Result<Vec
                 Some(bytes) => JSArg::Buffer(bytes.to_vec()),
                 None => JsonValue::Null.into(),
             },
-            (quaint_value @ quaint::ValueType::Numeric(bd), _) => match bd {
-                Some(bd) => match bd.to_string().parse::<f64>() {
-                    Ok(double) => JSArg::from(JsonValue::from(double)),
-                    Err(_) => JSArg::from(JsonValue::from(quaint_value.clone())),
-                },
+            (quaint::ValueType::Numeric(bd), _) => match bd {
+                Some(bd) => JSArg::RawString(bd.to_string()),
                 None => JsonValue::Null.into(),
             },
             (quaint::ValueType::Array(Some(items)), _) => JSArg::Array(values_to_js_args(items)?),
```

This also fixes this test, while changing the test itself seems to be only masking the issue, but I feel like I might be missing something.

Reply:
Yes, my thinking is that 10e-18 precisión was enough and the minimum guaranteed by any driver (rust or otherwise). After all the test was skipped on mongo for this lack of precision, having said that. I don't have a strong opinion, and I can't revert back to send a string over the rust-js boundaries rather than a double. And if the choice I made was unexpected to you that might be a good reason to do so.

Reply:
Addressed.